go.mod

```diff
@@ -1,10 +1,12 @@
 module github.com/coinbase-samples/advanced-trade-sdk-go
 
-go 1.19
+go 1.21
```

Comment:
Why different version, wouldn't it be better to just set the version to `go 1.24.5` and remove the `toolchain ...` line?